### PR TITLE
appveyor: bundle groonga-normalizer-mysql

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

**Comment by kenhys** [comment]
Thanks. @komainu8 

@kou could you review this PR, please?

File: appveyor.yml [hunk 1]
```diff
@@ -66,6 +66,10 @@ install:
         $Env:GROONGA_INSTALL_FOLDER += "-x64"
       }
       $Env:FULL_GROONGA_INSTALL_FOLDER = $Env:APPVEYOR_BUILD_FOLDER + "\" + $Env:GROONGA_INSTALL_SUB_FOLDER + "\" + $Env:GROONGA_INSTALL_FOLDER
+  - curl -O http://packages.groonga.org/source/groonga-normalizer-mysql/groonga-normalizer-mysql-latest.zip &&
+      7z x groonga-normalizer-mysql-latest.zip &&
+      del groonga-normalizer-mysql-latest.zip &&
+      move groonga-normalizer-mysql* vendor\plugins\groonga-normalizer-mysql
```
Comment (kou) [inline appveyor.yml]: Why did you use `&&` instead of multiple `-`?

Comment (kenhys) [reply appveyor.yml]: Oh, surely multiple `-` is enough. I've fixed it.

**Comment by kenhys** [comment]
@kou as #917 #918 was merged, I've rebased and fixed your feedback!

**Pull request merged**